doc/articles/getting-started-turtorial-1.md

```diff
@@ -0,0 +1,72 @@
+# Create a Single Page App with Uno
+
+[Download the complete sample](https://github.com/nventive/uno.GettingStartedTutorial)
+
+In this tutorial you will learn how to:
+
+- Add the Uno Platform Project Templates to Visual Studio
+- Create a new Project with Uno Platform
+- Learn basics on Model Binding
+
+The tutorial walks you through creating a cross platform application with Uno Platform - a simple Bug Tracking app - which enables you to see a single Issue entry.
+
+<div class="NOTE alert alert-info">
+<h5>Before you start the tutorial</h5>
+
+[If you haven't prepared your dev environment for Uno app development yet, the Setting up the Environment page will guide you through installing all the prerequisites.](get-started.md)
+
+</div>
+
+## Getting Started
+
+1. Open Visual Studio and click on `Create new project`. 
+
+    ![](Assets/tutorial01/newproject1.PNG)
+
+2. Search for the `Uno` templates, select the `Cross-Platform App (Uno Platform)` then click `Next`.
+
+    ![](Assets/tutorial01/newproject2.PNG)
+
+3. Name your app then click `Next`.
+
+4. In your `Shared` project, open up `MainPage.xaml`
+
+5. Add a `StackPanel` around your `TextBlock`
+``` xaml
+<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <StackPanel>
+		<TextBlock Text="Hello World" Margin="20" FontSize="30" />
+	</StackPanel>
```

Comment:
Might want to avoid mixing up space/tab: 
![image](https://user-images.githubusercontent.com/2359550/89092246-fd2a4500-d37d-11ea-9d40-dd5d4b4789d1.png)